test/acceptance/bytegroup/byte.go

```diff
@@ -0,0 +1,159 @@
+package bytegroup
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This comment block should go at the beginning of the file.

Reply:
Done